docs/section19/19-13.md

```diff
@@ -0,0 +1,75 @@
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Reset all_design_setpoints_delta_Ts_are_per_reqs boolean to true: `all_design_setpoints_delta_Ts_are_per_reqs = true`  
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in target_system_types):`      
+            - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+
+            Below determines the zone setpoint to use in the delta T calculation for systems with multiple zone thermostat setpoints, use the design set point that will result in the lowest supply air cooling set point or highest supply air heating set point: 
+            - Reset the design_thermostat_cooling_setpoint variable from the previous HVAC system: `design_thermostat_cooling_setpoint = +infinity`  
+            - Reset the design_thermostat_heating_setpoint variable from the previous HVAC system: `design_thermostat_heating_setpoint = -infinity`  
+            - Cycle through each zone associated with the HVAC system to get the lowest design_thermostat_cooling_setpoint and the highest design_thermostat_heating_setpoint: `for zone in zones_list_hvac_sys_b:`  
+                - Check if the zone_design_thermostat_cooling_setpoint is less than the design_thermostat_cooling_setpoint: `if zone.zone_design_thermostat_cooling_setpoint < design_thermostat_cooling_setpoint:` 
+                    - Set the design setpoint equal to the zone design setpoint: `design_thermostat_cooling_setpoint = zone.zone_design_thermostat_cooling`  
+                - Check if the zone_design_thermostat_heating_setpoint is greater than the design_thermostat_heating_setpoint: `if zone.zone_design_thermostat_heating_setpoint > design_thermostat_heating_setpoint:`  
+                    - Set the design setpoint equal to the zone design setpoint: `design_thermostat_heating_setpoint = zone.zone_design_thermostat_heating`  
+            
+            Below checks for a delta T of 20F:
+            - For each zone in the list of zones served by the hvac system: `for zone in zones_list_hvac_sys_b:`  
+                - Reset zone has lab space boolean variable: `zone_has_lab_space = false`  
```

Comment:
@claperle question: the section above determines the lowest cooling and the highest heating thermostat setpoint in the scenario where zones might have differing setpoints
The section line 51 onwards again cycles through each terminal of an HVAC system and calculates the delta. Shouldn't we be using the lowest cooling and the highest heating that was established through lines 42-49 for this calculation? If there are differing setpoints, then this would be correct. If there are not differing setpoints then also this would be correct.